### ISSUE TITLE
making :latest the "stable" branch

### ISSUE THREAD

**Issue by sparklyballs** [linuxserver/docker-unifi-controller#1]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]
	

##  Thanks, team linuxserver.io

+ unifi make great hardware but total crap software
+ making :latest , default tag makes it likely that upbreaks will occur whenever they vomit up a new release.
+ seeing as there is almost always never a downgrade option that is easy, effectively breaks people's installs.
+ plus you now unwittingly update EVERYONE using current latest , away from LTS.... 
+ most people are just gonna pull the latest everytime there is an update......

**Comment by thelamer** [comment]
this has been discussed to death on Discord, this is the decision the team made. 
I am indifferent, just applying what everyone agreed on. 
Anyone serious about this will be either: 
1. not auto updating their container
2. using a specific version tag not even one of the meta tags IE "5.10.12-ls1" just pushed.

**Comment by sparklyballs** [comment]
but making :latest as their supposedly stable branch, 95% of people are gonna pull latest whenever there is an update (i.e checking via unraid webui for updates) and you're gonna move them away from LTS... 

bad move, very bad move.

**Comment by JAGFin1** [comment]
This is a solution the team has discussed at quite some length over the last few weeks. With the pipeline migration almost finished, we needed to decide how best to approach the tagging system for UniFi, as tagging on our images is now more verbatim with regards to the application version. 

The LTS/Unstable/Latest issues have always been a thorn in our side but now is the best time to start attempting to rectify the issue. With that in mind, this is the decision we have made:

:latest will be the literal latest version of the controller. This is regardless of our opinions on the relative stability of the release. We tried in the past to make attempts to protect users from broken releases in the past, we got shit about it in return. 

:lts will be locked to the current LTS branch of releases. At time of comment, this is the 5.6.x release chain. Anyone wishing to remain on this can make use of this tag. 

This more logically follows the approach of semantic versioning and makes it a lot clearer what “latest” actually is: it is quite literally the latest and most recent highest version of the UniFi controller. 

One thing to keep in mind here is that this is a completely new image, separate from our existing UniFi image. We want this to have the new tagging nomenclature while the old one (for the time being) remains untouched, eventually being deprecated after a specific notice period. 

We collectively feel it is the best approach for this specific image as it helps align our general approach to versioned tagging.
